isis/src/base/apps/mosrange/mosrange.cpp

```diff
@@ -67,18 +69,42 @@ namespace Isis {
     return (localRadius / pixres * pi_c() / 180.0);
   }
 
-  void mosrange(UserInterface &ui, Pvl *log) {
-    Process p;
-
-    // Get the list of names of input CCD cubes to stitch together
-    FileList flist;
-    flist.read(ui.GetFileName("FROMLIST"));
-    if(flist.size() < 1) {
+  /**
+   * Compute lat/lon range of a set of camera images for mosaicking
+   *
+   * @param ui UserInterface object containing parameters
+   * @return Pvl results log file
+   *
+   * @throws IException::User "The list file [FILENAME] does not contain any filenames"
+   */
+  Pvl mosrange(UserInterface &ui) {
+
+    // Get the list of names of input cubes to stitch together
+    FileList cubeFileList;
+    cubeFileList.read(ui.GetFileName("FROMLIST"));
+    if ( cubeFileList.size() < 1)  {
       QString msg = "The list file[" + ui.GetFileName("FROMLIST") +
                     " does not contain any filenames";
       throw IException(IException::User, msg, _FILEINFO_);
     }
```

Comment:
Might be better to move this check into `Pvl mosrange(FileList &cubeFileList, UserInterface &ui)` so the length is checked even if a user calls either directly outside of an app context.

Reply:
Thanks a lot for having a look so quickly Kelvin. Just committed your suggested change.